manifests/c/ctrl-f/userdiag/0.1.1/ctrl-f.userdiag.locale.fr-FR.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: ctrl-f.userdiag
+PackageVersion: 0.1.1

```

```suggestion
PackageVersion: 0.4.97
```
You'll also need to update the version folder from `manifests/c/ctrl-f/userdiag/0.1.1` to `manifests/c/ctrl-f/userdiag/0.4.97`